lib/rucio/tests/test_abacus_account.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2018 CERN for the benefit of the ATLAS collaboration.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Hannes Hansen <hannes.jakob.hansen@cern.ch>, 2018
+
+from nose.tools import assert_equal
+
+import os
+
+from rucio.db.sqla import models
+from rucio.db.sqla.session import get_session
+from rucio.client.uploadclient import UploadClient
+from rucio.common.utils import generate_uuid
+from rucio.core.account_limit import get_account_usage, set_account_limit
+from rucio.core.rse import get_rse
+from rucio.daemons import undertaker
+from rucio.daemons.abacus import account
+from rucio.daemons.judge import cleaner
+from rucio.daemons.reaper import reaper
+from rucio.tests.common import file_generator
+
+
+class TestAbacusAccount():
+
+    def setUp(self):
+        self.account = 'root'
+        self.scope = 'mock'
+        self.upload_client = UploadClient()
+        self.file_sizes = 2
+        self.rse = 'MOCK4'
```

Comment:
use rse_name_generator()

Reply:
For this case I wanted to use one of the simulated RSEs where I can upload a real file. I am not sure if this is possible with a random RSE.

Reply:
understood